index.bs

```diff
@@ -1315,7 +1318,8 @@ credential.
     ::  The [=user handle=] of the user account entity.
 
     :   <dfn>displayName</dfn>
-    ::  A friendly name for the user account (e.g., "John P. Smith").
+    ::  A friendly name for the user account (e.g., "John P. Smith"). Authenticators while creating a credential MUST support
+        minimum of 64 bytes for this field and optionally can truncate how much it wants to store if this field is more than 64 bytes.
```

Comment:
suggested re-write:
[=Authenticators=] MUST support a 64 byte minimum length for a [=PublicKeyCredentialEntity/displayName=] members's value. Authenticators MAY truncate a [=PublicKeyCredentialEntity/displayName=] member's value to a length equal to or greater than 64 bytes.